src/pages/integrations/platforms/vercel.mdx

```diff
@@ -0,0 +1,380 @@
+
+2. Click on **Vercel**
+
+![Select Vercel](/assets/images/platform-integrations/vercel/1-vercel-create-new-service-credential.png)
+
+3. Paste your `Vercel Token` and add a descriptive name for the service credential and click **Save**
+
+![Add vercel token as service credential](/assets/images/platform-integrations/vercel/2-vercel-save-new-service-credentials.png)
+
+## Step 2: Set up a secret sync
+
+1. Go to your App in the Phase Console and go to the **Syncing** tab. Select **Vercel** under the 'Create a new Sync' menu.
+
+![Set up an vercel secret sync](/assets/images/platform-integrations/vercel/1-create-new-sync-select-vercel.png)
+
+2. Choose the credentials you added in the previous step as the authentication mode, and click **Next**.
+
+![Select service credentials for the sync](/assets/images/platform-integrations/vercel/2-create-new-sync-select-vercel-service-credentials.png)
+
+3. Next, configure the source and destination for your secrets. Pick an Environment from your App as the source. To configure the destination for the sync, choose a Vercel Project from the list of available Projects, as well as the Target Environment and the Secret Type.
+
+![Set up the sync config](/assets/images/platform-integrations/vercel/3-create-new-sync-choose-secret-config.png)
+
+4. Once set up, your secrets will automatically be synced to the chosen destination project in Vercel. You can manage your sync from the *Syncing* tab of your App, or from the *Integrations* screen.
+
+![Sync status](/assets/images/platform-integrations/vercel/4-create-new-sync-sync-status-success.png)
+
+## Reserved environment variables keys by Vercel
+
+The following secret keys are reserved or prepopulated by Vercel for technical reasons on their end that Phase will automatically filter for and not sync with Vercel to avoid errors.
+
+### Environment variables that are reserved by Vercel Serverless Function runtimes that Phase will filter for. 
+
+- `AWS_SECRET_KEY`
+- `AWS_EXECUTION_ENV`
+- `AWS_LAMBDA_LOG_GROUP_NAME`
+- `AWS_LAMBDA_LOG_STREAM_NAME`
+- `AWS_LAMBDA_FUNCTION_NAME`
+- `AWS_LAMBDA_FUNCTION_MEMORY_SIZE`
+- `AWS_LAMBDA_FUNCTION_VERSION`
+- `NOW_REGION`
+- `TZ`
+- `LAMBDA_TASK_ROOT`
+- `LAMBDA_RUNTIME_DIR`
+
+### Environment Variables are allowed by Vercel Serverless Function runtimes that Phase will not filter for:
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_SESSION_TOKEN`
+- `AWS_REGION`
+- `AWS_DEFAULT_REGION`
+
+<div className="not-prose">
+  <Button
+    href="https://vercel.com/docs/projects/environment-variables/reserved-environment-variables"
+    variant="text"
+    arrow="right"
+    children="Reserved Environment Variables - Vercel Docs"
+  />
+</div>
+
+### System environment variables:
+
+Vercel provides a set of Environment Variables that are automatically populated by the System, such as the URL of the Deployment or the name of the Git branch deployed.
```

Comment:
Its not entirely clear what the user should do with this information. Does this mean env vars in Phase with these names will be filtered out? Will Vercel throw an error? Will they be overwritten and this is just a warning?